docs/clients/athena.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: client
+category: clients
+name: Athena
+package: async-aws/athena
+---
+
+## Usage
+
+### List Databases
+
+```php
+use AsyncAws\Athena\AthenaClient;
+use AsyncAws\Athena\Input\ListDatabasesInput;
+
+$athena = new AthenaClient();
+
+$result = $athena->listDatabases(new ListDatabasesInput([
+    'CatalogName' => 'my_catalog'
+    'NextToken' => 'next-1045'
+    'MaxResults' => 20
```

Comment:
nextToken is used by AWS internally for pagination. This is automatically handled by `async-aws` we should remove it from the documentation

```
$result = $athena->listDatabases(['CatalogName' => 'my_catalog']);
foreach ($result->getDatabaseList() as $database) {
...
```